src/overlays/actors/ovl_En_Invadepoh/z_en_invadepoh.c

```diff
@@ -17,495 +242,5248 @@ ActorInit En_Invadepoh_InitVars = {
+    this->modelInfo.headRotStepScale = 0.08f;
+    this->modelInfo.headRotMaxStep = 0x7D0;
+    Animation_MorphToLoop(&this->skelAnime, &gCremiaIdleAnim, -6.0f);
+    this->shapeAngularVelocityY = 0;
+    this->actionFunc = EnInvadepoh_Night3Cremia_Idle;
+}
+
+/**
+ * Rotates Cremia to face the player and waits until Romani is done talking, at which point Cremia will walk again.
+ */
+void EnInvadepoh_Night3Cremia_Idle(EnInvadepoh* this, PlayState* play) {
+    EnInvadepohModelInfo* modelInfo = &this->modelInfo;
+    Player* player = GET_PLAYER(play);
+    s16 pitch;
+    s16 yaw;
+
+    Math_StepToS(&this->shapeAngularVelocityY, 0x7D0, 0xC8);
+    Math_SmoothStepToS(&this->actor.shape.rot.y, this->actor.yawTowardsPlayer, 6, this->shapeAngularVelocityY, 0x28);
+
+    pitch = (s16)(Math_Vec3f_Pitch(&this->actor.focus.pos, &player->actor.focus.pos) * 0.85f) - this->actor.shape.rot.x;
+    modelInfo->headRotTarget.x = CLAMP(pitch, -0xBB8, 0xBB8);
+
+    yaw = this->actor.yawTowardsPlayer - this->actor.shape.rot.y;
+    modelInfo->headRotTarget.y = CLAMP((s16)(yaw * 0.7f), -0x1F40, 0x1F40);
+
+    if (sNight3Romani == NULL) {
+        EnInvadepoh_Night3Cremia_SetupWalk(this);
+    } else if ((sNight3Romani != NULL) && (sNight3Romani->actionFunc != EnInvadepoh_Night3Romani_Talk)) {
+        EnInvadepoh_Night3Cremia_SetupWalk(this);
+    }
+}
+
+void EnInvadepoh_Night3Cremia_WaitForObject(Actor* thisx, PlayState* play2) {
+    PlayState* play = play2;
+    EnInvadepoh* this = THIS;
+    s32 pad;
+
+    if (Object_IsLoaded(&play2->objectCtx, this->objectSlot)) {
+        s32 currentTime = CURRENT_TIME;
+
+        this->actor.objectSlot = this->objectSlot;
+        Actor_SetObjectDependency(play, &this->actor);
+        EnInvadepoh_Cremia_DesegmentTextures();
+        SkelAnime_InitFlex(play, &this->skelAnime, &gCremiaSkel, &gCremiaWalkAnim, this->jointTable, this->morphTable,
+                           CREMIA_LIMB_MAX);
+        EnInvadepoh_ModelInfo_Init(&this->modelInfo, sCremiaEyeAnimations, CREMIA_EYE_ANIM_OPEN_THEN_RANDOM_BLINK,
+                                   sCremiaMouthAnimations, CREMIA_MOUTH_ANIM_NORMAL, &gZeroVec3s, 100, 0.03f, 0.3f,
+                                   0.03f);
+        this->actor.textId = 0x33CD;
+
+        if (currentTime < CLOCK_TIME(20, 01) + 30) {
+            this->angle = -0x8000;
+            this->distanceToRomani = 40.0f;
+        } else if (currentTime > CLOCK_TIME(20, 14) + 1) {
+            this->angle = -0x4800;
+            this->distanceToRomani = 20.0f;
+        } else {
+            this->angle = -0x8000;
+            this->distanceToRomani = 40.0f;
+        }
+
+        if ((CLOCK_TIME(6, 00) <= currentTime) && (currentTime < CLOCK_TIME(20, 00) + 30)) {
+            this->actor.update = EnInvadepoh_Night3Cremia_WaitForTime;
+            this->actor.draw = NULL;
+        } else if ((CLOCK_TIME(20, 00) + 30 <= currentTime) && (currentTime < CLOCK_TIME(20, 15))) {
```

Comment:
```suggestion
        if ((currentTime >= CLOCK_TIME(6, 00)) && (currentTime < CLOCK_TIME(20, 00) + 30)) {
            this->actor.update = EnInvadepoh_Night3Cremia_WaitForTime;
            this->actor.draw = NULL;
        } else if ((currentTime >= CLOCK_TIME(20, 00) + 30) && (currentTime < CLOCK_TIME(20, 15))) {
```
I think this is more common?